packages/slugify/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/benlinton/slugify
+TERMUX_PKG_DESCRIPTION="Bash command that converts filenames and directories to a web friendly format."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE.git
+TERMUX_PKG_VERSION=1.0.1
+TERMUX_PKG_GIT_BRANCH=master
```

Comment:
These two lines are inconsistent with each other. `master` is not version `1.0.1`.